Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
I understand that homebrew won't create total .app bundles. These are for download and install.

To clarify: It's not an .app bundle containing dylibs, etc. It is simply the folder structure with themost essential files (icon, plist and starter script) to make a clickable icon in the Laucher by using "brew linkapps ..." The scripts simply create this folder structure. I talked about putting such a routine into this script with Mike (see #43677). Maybe I got him wrong for language/translation reasons. Please tell me...

When I got that right (I'm no native english speaker; "Certainly not be default at least."): Adding an option to the formula like '--with-appstarter' would be okay and you might accept the formula?